Dockerfile

```diff
@@ -7,9 +8,14 @@ RUN apk --update add git
 
 RUN npm install
 
-RUN adduser -u 9000 -D app
-USER app
+
+RUN apk del --purge git
```

Comment:
Interesting: I didn't realize that. Makes sense when you think it through, though. Will update.